install/kubernetes/values.yaml

```diff
@@ -171,6 +171,11 @@ tetragonOperator:
     # -- Enables the PodInfo CRD and the controller that reconciles PodInfo
     # custom resources.
     enabled: false
+  # -- Enables the Tetragon Operator metrics.
+  prometheus:
+    enabled: true
```

Comment:
I see this value doesn't affect whether the operator runs the metrics server or not, only whether the k8s service is created or not. It would be nice to propagate it into the operator code too, and don't run the metrics server if it's set to false. But I think this can be done as a follow-up, not necessarily in this PR.

Reply:
um agreed, this be done as follow-up, might update the comment in values to be easier track at that level though